### PR TITLE
Images

### PR DESCRIPTION
**Pull request by itamargreen**
I'm suspicious that this doesn't break anything.

### CODE REVIEW

File: client/app/modals/image-upload/image-upload.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <div class="dimmer">
+    <div class="big loader"></div>
+  </div>
+
+  <file-drop class="expanded button" [ngClass]="{ 'hollow': !fileHovering }" (onFileOver)="fileOver($event)" (onFileLeave)="fileLeave($event)" (onFileDrop)="dropped($event)">Drop your image file here</file-drop>
+
+<div *ngIf="image">
+  <img src={{this.image.image}} />
+</div>
+  <!--<div class="grid-x align-center button-group">-->
```
Comment (taltaub22) [inline client/app/modals/image-upload/image-upload.component.html]: Please delete this commented code

File: client/app/modals/image-upload/image-upload.component.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          this.loading = false;
+        })
+      })
+    } else {
+      this.file = null
+    }
+  }
+
+  //
+  // public upload(event) {
```
Comment (taltaub22) [inline client/app/modals/image-upload/image-upload.component.ts]: Please delete this commented code

File: client/app/services/images.service.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    if (!this.initStarted) {
+      this.initStarted = true
+      this.reload()
+    }
+  }
+
+  reload () {
+    return this.requests.get('/image/all').subscribe((images: Image[]) => {
+      this.images = images.map(image => new Image().deserialize(image))
+      //TODO: Add error handling
```
Comment (taltaub22) [inline client/app/services/images.service.ts]: That's the time to add error handling...

File: routers/imagesRouter.js [hunk 4]
```diff
@@ -34,6 +34,21 @@ router.get('/:imageName', (req, res) => {
   })
 })
 
+router.post('/upload', adminAction, (req, res) => {
```
Comment (taltaub22) [inline routers/imagesRouter.js]: Please explain what you did here

Comment (itamargreen) [reply routers/imagesRouter.js]: I didn't notice that the existing post was supposed to do this. Should I combine them?

**Comment by taltaub22** [comment]
All good, fixed a bug that I had.
You can merge, please remember to delete this branch afterwords

**Pull request merged**